docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,162 @@ Notice that this time, the `uname -a` output shows `armv7l` and
 
 Multi-architecture support makes it easy to build <a href="https://blog.docker.com/2017/11/multi-arch-all-the-things/" target="_blank">multi-architecture Docker images</a> or experiment with ARM images and binaries from your Mac.
 
+## Buildx Tech Preview
+
+With the Docker Desktop 2.0.4.0 Edge release, Docker is making it easier than ever to develop containers on, and for Arm servers and devices. Using the standard Docker tooling and processes, you can start to build, push, pull, and run images seamlessly on different compute architectures. Note that you don't have to make any changes to Dockerfiles or source code to start building for Arm.
```

Comment:
Before merging, could you wrap the document to 80 chars?

Reply:
@thaJeztah - The output renders neatly on Netlify -https://deploy-preview-8818--docsdocker.netlify.com/docker-for-mac/multi-arch/#buildx-tech-preview

 Do I still need to wrap this to 80 characters? Also, is this something we should follow across all docs? Thanks.

Reply:
It won't make a difference in the produced HTML, but we standardized on 80 characters for Markdown, as it helps when reviewing (eg you can point to a specific line, instead of "somewhere in this paragraph there's a typo")

in addition, it makes the source better readable and can reduce the diff in come cases and  the risk of conflicts when multiple PRs modify the same file